crates/cast/src/cmd/erc20.rs

```diff
@@ -19,10 +19,17 @@ sol! {
     #[sol(rpc)]
     interface IERC20 {
         #[derive(Debug)]
+        function name() external view returns (string);
+        function symbol() external view returns (string);
+        function decimals() external view returns (uint8);
+        function totalSupply() external view returns (uint256);
         function balanceOf(address owner) external view returns (uint256);
         function transfer(address to, uint256 amount) external returns (bool);
         function approve(address spender, uint256 amount) external returns (bool);
         function allowance(address owner, address spender) external view returns (uint256);
+        function transferFrom(address from, address to, uint256 amount) external returns (bool);
```

Comment:
idt this one make sense, iirc it would mean you had approved an eoa to pull funds from you, i've never seen that before